### PR TITLE
Reference documentation for the Window Controls Overlay API

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary
This PR adds the required documentation for the WindowControlsOverlay JS API.

#### Motivation
Desktop-installed PWAs can now use the Window Controls Overlay feature (available generally starting from Chrome/Edge 98).

Using this feature requires using the `window-controls-overlay` display mode in the `display_override` member, which was already documented here: https://developer.mozilla.org/en-US/docs/Web/Manifest/display_override

And the feature also relies on new CSS environment variables (`titlebar-area-*`) which were also already documented here: https://developer.mozilla.org/en-US/docs/Web/CSS/env()#using_env_to_ensure_content_is_not_obscured_by_window_control_buttons_in_desktop_pwas

The remaining piece for this feature is a JavaScript API: `navigator.windowControlsOverlay` which I am documenting in this PR.
The API isn't very complicated, it has just one property (`visible`), one event handler (`ongeometrychange`), and one method (`getTitlebarAreaRect`).

I believe I have made all of the necessary changes for these things to be documented.

**Notes**

* I did not create a separate interface reference page for the WindowControlsOverlayGeometryChangeEvent. This event is (for now) simple enough that I documented its properties in the event page, instead of creating another one for the interface.
* The version of BCD currently in use needs to be updated to fix the current BCD errors. I am adding the necessary compat data in the BCD repo here: https://github.com/mdn/browser-compat-data/pull/14095.

#### Related issues
This is part of this content proposal parent issue: https://github.com/mdn/content/issues/10539

#### Metadata
- [x] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      <td>
+        <code
+          ><a href="/en-US/docs/Web/API/WindowControlsOverlay/ongeometrychange"
+            >ongeometrychange</a
+          ></code
+        >
+      </td>
+    </tr>
+  </tbody>
+</table>
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:37]:
```suggestion
## Syntax

Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.

```js
addEventListener('geometrychange', (event) => { });

geometrychange = (event) => { });
```

```

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The `geometrychange` event has the following properties:
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:39]:
```suggestion
## Event type

A {{domxref("GeometryChangeEvent")}}. Inherits from {{domxref("[Event")}}.

{{InheritanceDiagram(500, 70, 50, "GeometryChangeEvent")}}

## Event properties

_In addition to the properties listed below, properties from the parent interface, {{domxref("Event")}}, are available._
```

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+- `titlebarAreaRect`
+  - : A {{domxref("DOMRect")}} representing the position and size of the title bar area.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:42]:
```suggestion
  - : A {{domxref("DOMRect")}} representing the position and size of the title bar.
```

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+title: 'WindowControlsOverlay: geometrychange event'
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:2]: @tomayac Chrome's IDL says `WindowControlsOverlayGeometryChangeEvent` (as does your article). I'm guessing one of these is out of date, but which?

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md (hunk 1 above)
Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:37]: Don't commit this until we nail down what the event name is going to be. We may need to change it.

File: files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  - API
+  - Method
+  - Reference
+  - WindowControlsOverlay
+  - Progressive Web Apps
+browser-compat: api.WindowControlsOverlay.getTitleBarAreaRect
+---
+{{APIRef("Window Controls Overlay API")}}{{SeeCompatTable}}
+
+The **`WindowControlsOverlay.prototype.getTitleBarAreaRect()`** method queries the current geometry of the title bar area of the desktop Progressive Web App window.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md:14]:
```suggestion
The **`getTitleBarAreaRect()`** method of the {{domxref('WindowControlsOverlay')}} interface queries the current geometry of the title bar area of a desktop Progressive Web App window.
```

File: files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Syntax
+
+```js
+navigator.windowControlsOverlay.getTitleBarAreaRect();
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md:19]:
```suggestion
getTitleBarAreaRect();
```

Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md:19]: Per the [current instructions](https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods).

File: files/en-us/web/api/windowcontrolsoverlay/index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+### Event handlers
+
+- {{domxref("WindowControlsOverlay.onGeometryChange")}}
+  - : A handler for the {{domxref("WindowControlsOverlay/geometrychange_event", "geometrychange")}} event. This event is sent when the geometry of the title bar area changes.
+
+## Methods
+
+- {{domxref("WindowControlsOverlay.getTitlebarAreaRect()")}}
+  - : Returns the size and position of the title bar area.
+
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/index.md:32]:
```suggestion
## Events
Listen to these events using {{domxref('EventTarget.addEventListener()')}} or by assigning an event listener to the relevant `oneventname` property of this interface.

- [`geometrychange`](/en-US/docs/Web/API/VisualViewport/geometrychange_event)
  - : Fired when the geometry of the title bar area changes.
  
```

File: files/en-us/web/api/windowcontrolsoverlay/index.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+## Properties
+
+- {{domxref("WindowControlsOverlay.visible")}} {{readonlyInline}}
+  - : A {{Glossary("Boolean")}} that indicates whether the window controls overlay is visible or not.
+
+### Event handlers
+
+- {{domxref("WindowControlsOverlay.onGeometryChange")}}
+  - : A handler for the {{domxref("WindowControlsOverlay/geometrychange_event", "geometrychange")}} event. This event is sent when the geometry of the title bar area changes.
+
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/index.md:27]:
```suggestion
```

Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlay/index.md:27]: This is moved down and done in a different way.

File: files/en-us/web/api/windowcontrolsoverlay/index.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+## Methods
+
+- {{domxref("WindowControlsOverlay.getTitlebarAreaRect()")}}
+  - : Returns the size and position of the title bar area.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/index.md:31]:
```suggestion
  - : Returns the size and position of the title bar.
```

File: files/en-us/web/api/windowcontrolsoverlay/ongeometrychange/index.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/ongeometrychange/index.md:1]: Please delete this whole page.

File: files/en-us/web/api/windowcontrolsoverlay/visible/index.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+  - Reference
+  - WindowControlsOverlay
+  - Progressive Web Apps
+browser-compat: api.WindowControlsOverlay.visible
+---
+{{ApiRef("Window Controls Overlay API")}}
+
+The **`visible`** property of a {{domxref("WindowControlsOverlay")}} object returns a {{Glossary("Boolean")}} that indicates whether the window controls overlay is visible or not.
+
+The window controls overlay will not be visible if the Progressive Web App does not use the Window Controls Overlay feature at all, or if the user has opted-out of the feature.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/visible/index.md:17]: >does not use the Window Controls Overlay feature at all

I would rephrase this in terms of Web App Manifest configuration with a link. The nature of reference is such that you should never assume a reading order.

File: files/en-us/web/css/env()/index.md [hunk 12]
```diff
@@ -172,7 +172,7 @@ The syntax of the fallback, like that of custom properties, allows commas. But,
 
 ### Using env() to ensure content is not obscured by window control buttons in desktop PWAs
 
-In the following example `env()` is used to ensure that content displayed in a PWA that uses the Window Controls Overlay feature is not obscured by the operating system's window control buttons. The `titlebar-area-*` values define a rectangle where the title bar would normally have been displayed. On devices that do not support the Window Controls Overlay feature, such as mobile devices, the fallback values are used.
+In the following example `env()` is used to ensure that content displayed in a desktop Progressive Web App that uses the [Window Controls Overlay API](/en-US/docs/Web/API/Window_Controls_Overlay_API) is not obscured by the operating system's window control buttons. The `titlebar-area-*` values define a rectangle where the title bar would normally have been displayed. On devices that do not support the Window Controls Overlay feature, such as mobile devices, the fallback values are used.
```
Comment (jpmedley) [inline files/en-us/web/css/env()/index.md:175]:
```suggestion
In the following example `env()` ensures that content displayed in a desktop Progressive Web App that uses the [Window Controls Overlay API](/en-US/docs/Web/API/Window_Controls_Overlay_API) is not obscured by the operating system's window control buttons. The `titlebar-area-*` values define a rectangle where the title bar would normally have been displayed. On devices that do not support the Window Controls Overlay feature, such as mobile devices, the fallback values are used.
```

File: files/jsondata/GroupData.json [hunk 13]
```diff
@@ -2187,6 +2187,15 @@
         "XRSession: visibilitychange"
       ]
     },
+    "Window Controls Overlay API": {
```
Comment (jpmedley) [inline files/jsondata/GroupData.json:2190]: @Elchi3 Can you help me out with a review of GroupData. I'm rusty.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/navigator/windowcontrolsoverlay/index.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+the title bar area, and know whenever it changes, in order to position web content in
+that area.
+
+## Syntax
+
+```js
+var wco = navigator.windowControlsOverlay
+```
+
+### Value
```
[the remark this answers is not in the document]
Comment (tomayac) [reply files/en-us/web/api/navigator/windowcontrolsoverlay/index.md:35]: This should also use modern `const` and `let`.

[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/navigator/windowcontrolsoverlay/index.md:35]: Thanks. MDN decided not to show syntax for properties any more. They just show the return value, which is what developers need to know. Properties all have the same syntax and it's something that doesn't need to be explained.

File: files/jsondata/GroupData.json (hunk 13 above)
Comment (Elchi3) [reply files/jsondata/GroupData.json:2190]: Looks good to me. 
I mean there isn't anything to group here really, I guess.

File: files/jsondata/GroupData.json [hunk 15]
```diff
@@ -2187,6 +2187,15 @@
...
+      "overview": ["Window Controls Overlay API"],
+      "interfaces": [
+        "WindowControlsOverlay"
+      ],
+      "methods": [],
+      "properties": [],
```
Comment (Elchi3) [inline files/jsondata/GroupData.json:2196]: This could be added: 

```suggestion
      "properties": ["Navigator.windowControlsOverlay"],
```

Comment (jpmedley) [reply files/jsondata/GroupData.json:2196]: @captainbrosset, please commit this.

File: files/en-us/web/api/window_controls_overlay_api/index.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+## Interfaces
+
+- {{domxref("WindowControlsOverlay")}}
+  - : Provides information about the visibility and geometry of the title bar area and an event to know whenever it changes.
+
+## Specifications
+
+| Specification                         | Status                             | Comment             |
+| ------------------------------------- | ---------------------------------- | ------------------- |
+| {{SpecName("WindowControlsOverlay")}} | {{Spec2("WindowControlsOverlay")}} | Initial definition. |
```
Comment (hamishwillee) [inline files/en-us/web/api/window_controls_overlay_api/index.md:58]: Spec should generally be defined in browser compatibility under the key `api.WindowControlsOverlay` and accessed using `{{Specifications}}` macro. Unless there is a reason you have chosen not to of course :-).

```suggestion
{{Specifications}}
```

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md (hunk 2 above)
Comment (captainbrosset) [reply files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:39]: @jpmedley I'm assuming that means I need to create a GeometryChangeEvent interface doc as well, is that right?

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md (hunk 4 above)
Comment (captainbrosset) [reply files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:2]: @diekus could you shade some light here please?

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md (hunk 2 above)
Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md]: Yes.

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md (hunk 4 above)
Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:2]: Pinging this thread so we can move this forward. We appreciate the help you folks can give.

**Comment by captainbrosset** [comment]
I just pushed another commit to add the documentation for the `WindowControlsOverlayGeometryChangeEvent` interface.
Some notes about this:

* I previously called it `GeometryChangeEvent`. This was wrong, and I now corrected it to `WindowControlsOverlayGeometryChangeEvent`.
* I tried to clarify some of the previous pages I created too. This concept isn't very straightforward, and so I added a couple sentences on various pages mostly to link people back to the Window Controls Overlay API page when possible. What I feel is the most confusing is that the `titlebarAreaRect` stuff doesn't tell users about the title bar, but rather the area that it would occupy within the web content, if it had not been hidden with the WindowControlsOverlay feature.
* I still have an outstanding BCD pull request that needs to be merged and dependencies to be updated here for some of the spec/compat macros to stop failing: https://github.com/mdn/browser-compat-data/pull/14095

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/titlebararearect/index.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+The
+**`WindowControlsOverlayGeometryChangeEvent.titlebarAreaRect`** read-only property is a
+{{domxref("DOMRect")}} representing the position and size of the area occupied by the title bar in a desktop-installed Progressive Web App.
+
+## Syntax
+
+```js
+rect = WindowControlsOverlayGeometryChangeEvent.titlebarAreaRect
+```
+
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/titlebararearect/index.md:21]:
```suggestion
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md [hunk 18]
```diff
@@ -0,0 +1,34 @@
...
+tags:
+  - API
+  - WindowControlsOverlayGeometryChangeEvent
+  - Property
+browser-compat: api.WindowControlsOverlayGeometryChangeEvent.visible
+---
+{{APIRef("DOM Events")}}
+
+The
+**`WindowControlsOverlayGeometryChangeEvent.visible`** read-only property is a
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md:13]:
```suggestion
**`WindowControlsOverlayGeometryChangeEvent.visible`** read-only property of the {{domxref("WindowControlsOverlayGeometryChangeEvent")}} is a
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md [hunk 19]
```diff
@@ -0,0 +1,34 @@
...
+boolean flag that indicates whether the window controls overlay is visible or not in a desktop-installed Progressive Web App.
+
+If the user opts-out of the [Window Controls Overlay](/en-US/docs/Web/API/Window_Controls_Overlay_API) feature, the default title bar appears and the window controls buttons does not appear as an overlay, in which case the `visible` flag is false.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md:16]:
```suggestion
If the user opts-out of the [Window Controls Overlay](/en-US/docs/Web/API/Window_Controls_Overlay_API) feature, the default title bar appears and the window controls buttons do not appear as an overlay, in which case the `visible` flag is false.
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+
+## Syntax
+
+```js
+flag = WindowControlsOverlayGeometryChangeEvent.visible
+```
+
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md:23]:
```suggestion
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md [hunk 21]
```diff
@@ -0,0 +1,34 @@
...
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- The {{domxref("WindowControlsOverlayGeometryChangeEvent")}} interface it belongs to.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/visible/index.md:34]:
```suggestion
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/titlebararearect/index.md [hunk 22]
```diff
@@ -0,0 +1,32 @@
...
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- The {{domxref("WindowControlsOverlayGeometryChangeEvent")}} interface it belongs to.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/titlebararearect/index.md:32]:
```suggestion
```

Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/titlebararearect/index.md:32]: Because it's in the intro.

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+new WindowControlsOverlayGeometryChangeEvent(type, {visible: aBooleanValue, titlebarAreaRect: aDOMRectValue});
+```
+
+### Arguments
+
+_The `WindowControlsOverlayGeometryChangeEvent()` constructor also inherits arguments from
+{{domxref("Event.Event", "Event()")}}._
+
+- _type_
+  - : Is a {{domxref("DOMString")}} representing the name of the type of the
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md:28]:
```suggestion
  - : A {{domxref("DOMString")}} indicating the event type, always 
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md [hunk 24]
```diff
@@ -0,0 +1,45 @@
...
+    `WindowControlsOverlayGeometryChangeEvent`. It is case-sensitive.
+- `visible`
+  - : Is a boolean flag that's true when the titleBarAreaRect's values are not 0.
+- `titleBarAreaRect`
+  - : Is a {{domxref("DOMRect")}} representing the position and size of the title bar area.
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md:33]:
```suggestion
- `options`
  - : An object with the following properties:
  - `visible`
    - : A boolean flag that's true when the `titleBarAreaRect` object's values are not 0.
  - `titleBarAreaRect`
    - : A {{domxref("DOMRect")}} representing the position and size of the title bar area.
```

File: files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md [hunk 25]
```diff
@@ -0,0 +1,34 @@
+---
+title: WindowControlsOverlay.prototype.getTitleBarAreaRect()
```
Comment (jpmedley) [inline files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md:2]:
```suggestion
title: WindowControlsOverlay.getTitleBarAreaRect()
```

Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md:2]: I apologize for net catching this on the first review. We don't do this any more because 'prototype' wont' appear in a script. If there's a reason to distinguish between static and instance methods on a particular interface it's added to the basic description. For example, "The doStuff() static method of the whatever interface...".

**Review by jpmedley — CHANGES_REQUESTED** [review]
I appreciate your patience, and thank you for adding the additional pages. I know this is a difficult PR.

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md (hunk 23 above)
Comment (captainbrosset) [reply files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md:28]: I'm not sure I follow what you mean by `always`?
The value of this string is `change` when the event is fired. I don't know if any other string values are permitted when constructing the event though.

Comment (captainbrosset) [reply files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md]: I ended up removing the rest of the sentence, and only keeping `A {{domxref("DOMString")}} indicating the event type.` Is that ok?

Comment (jpmedley) [reply files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md]: What I'm trying to convey is that the exact type name of the event must be supplied in that parameter, and that that name is case sensitive. I also want to save readers from guessing. The spec doesn't make this clear because spec implementors are expected to know the idiom.

**Comment by captainbrosset** [comment]
Thanks for the reviews so far @jpmedley. I think I've addressed all comments. Let me know what you think.
Ideally, this PR would be ready before the WCO feature becomes available to all in stable versions of chromium browsers: beginning of February for Chrome and Edge.

File: files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md (hunk 4 above)
Comment (captainbrosset) [reply files/en-us/web/api/windowcontrolsoverlay/geometrychange_event/index.md:2]: The event name you are supposed to use in JS to listen to WCO geometry changes is: geometrychange. Example:

```javascript
navigator.windowControlsOverlay.addEventListener('geometrychange', () => {});
```

or 

```javascript
navigator.windowControlsOverlay.ongeometrychange = () => {};
```

Which is why I named this page `geometrychange_event/index.md`. Can you explain where `WindowControlsOverlayGeometryChangeEvent` would be used on this page?

**Review by jpmedley — APPROVED** [review]
Thank you for your patience.

**Comment by captainbrosset** [comment]
Thanks @jpmedley. Can this be merged then?

**Comment by captainbrosset** [comment]
Thank you Joe!

Just for reference, the following 2 BCD PRs have also been merged and we will need to update MDN's dependency on BCD to get the proper spec/compat-data information on the new pages added here:

https://github.com/mdn/browser-compat-data/pull/14035
https://github.com/mdn/browser-compat-data/pull/14095

**Comment by jpmedley** [comment]
BCD data is typically deployed once a week. It wasn't after these merges because of the holidays, but there was one yesterday. These should be on your page now.
